src/libs/ajax/billing/billing-models.ts

```diff
@@ -102,6 +102,8 @@ export interface WorkspaceSpendData {
   credits: string;
   currency: string;
   googleProjectId: string;
+  endTime?: string;
```

Comment:
I had included `endTime` and `startTime` in my test objects because that's what the real-life `WorkspaceSpendData` responses included, and it broke the build.  It seems like it makes sense to match the actual object, so I've added these fields in as optional